.gitignore

```diff
@@ -4,3 +4,4 @@
 .Ruserdata
 public
 blogdown
+_site
```

Comment:
This site is built by Hugo instead of Jekyll, and you don't need to ignore it. Hugo's default publish dir is `public`, which I have already ignored above.